doc/enterprise/index.md

```diff
@@ -0,0 +1,68 @@
+# Examples
```

Comment:
why are examples under "Enterprise" section?

Reply:
They should not be =)